packages/forms/src/components/ErrorSummary.tsx

```diff
@@ -94,8 +98,8 @@ interface ErrorSummaryProps {
   show: boolean;
 }
 
-const a = (chunks: React.ReactNode) => (
-  <Link external href="mailto:gctalent-talentgc@support-soutien.gc.ca">
+const supportLink = (chunks: React.ReactNode, locale: string) => (
+  <Link href={`/${locale}/support`} state={{ referrer: window.location.href }}>
```

Comment:
Shouldn't this be a new tab?

I feel like if you are contacting support regarding the form errors, you'd probably want the form errors open in one tab or window and the support form open in another. Easier to transcribe errors into the error ticket.

Reply:
I'm personally not a fan of forcing new tab links but here, I think [you have a good point](https://www.w3.org/TR/WCAG20-TECHS/G200.html#:~:text=An%20online%20form%20provides%20extensive%20context%2Dsensitive%20help%20for%20each%20form%20field%20on%20a%20separate%20page%20because%20there%20is%20too%20much%20text%20to%20include%20within%20the%20form.%20The%20links%20to%20the%20context%2Dsensitive%20help%20open%20in%20new%20windows%20or%20tabs%20to%20prevent%20the%20loss%20of%20any%20form%20data%20that%20has%20already%20been%20entered.). 👍 

Adjusted in [open support link in new tab](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8351/commits/b14429254673d0f3f90798a96e4695162cf63069)